Team07_CS555_Project.py

```diff
@@ -185,3 +209,11 @@ def display_gedcom_table(individuals, family):
 
         # Print The details using Pretty Table Library
         display_gedcom_table(individuals, family)
+
+        #User Story: 03 - Birth before death
+        Error03 = US3_birth_before_death(individuals)
+        print("Errors related to Birth before Death: ", Error03)
+
+        #User Story: 03 - Birth before death

```

Comment:
comment should be:
User story: 04 Marriage Before Divorce.

Reply:
I just copied the user story from Excel sheet.